spec.go

```diff
@@ -25,6 +25,10 @@ const (
 	StatusSuperseded = "superseded"
 	// StatusFailed indicates that the release was not successfully deployed.
 	StatusFailed = "failed"
+	// StatusNotInstalled indicates that an error occurred and Helm could not
+	// create a release. NOTE: This status is Giant Swarm specific and DOES
+	// NOT map to a Helm status.
+	StatusNotInstalled = "not installed"
```

Comment:
Out of curiosity, what is different from `failed` status?

Reply:
Yes I wasn't sure about this. So I went with keeping it as a constant in chart-operator.

Failed is helm created the release but there is a problem with it. Not installed is helm couldn't create a release. 

So it's not a true helm status which is why I left it out of helmclient. Let me know if you think it should be in helmclient.